openy.profile

```diff
@@ -180,6 +180,7 @@ function openy_demo_content_configs_map($key = NULL) {
       'openy_demo_bsimple',
       'openy_demo_bamenities',
       'openy_demo_tfitness',
+      'y_lb_init_demo_content'
```

Comment:
@podarok this one needs to be replaced with a new migration group `y_lb_init_demo_content`

Reply:
))

![image](https://github.com/YCloudYUSA/yusaopeny/assets/563412/b47b769c-4c87-4e03-9259-937d1b019299)


@aleevas

Reply:
@podarok sorry my bad, its already ok